.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+updates:
+  - directory: /
+    open-pull-requests-limit: 3
```

Comment:
Following the OpenSearch [conventions](https://github.com/opensearch-project/OpenSearch/blob/main/.github/dependabot.yml#L3) can we keep it to 1? In the future, with more dependencies in SDK this can be a pain.

Reply:
The limit seems to be 3 for [all the clients](https://github.com/search?q=org%3Aopensearch-project+open-pull-requests-limit&type=code) which is why I chose it.  

A lower number hides potential updates, I really don't see a need to limit it at all (the default is 5).

And ultimately I want to remove it and install Renovate, anyway...